### ISSUE TITLE
[Forms and Validation - Settings - Project Settings]Label is not defined for ‘Name edit field’ on project settings pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2565]
**User Experience:**
Users who rely on screen reader would find it difficult to get the appropriate information about various input fields.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Settings control lying in the header section on the page and select it.
4-Navigate to the Project Settings menu option listed under the Settings menu control and select it.
5-Navigate to the various controls lying in the pane opened.
6-Verify label is defined for edit field or not on project settings pane.

**Actual Result:**
Label is not defined for name edit field on project settings pane. Due to this narrator is reading only editing on name edit field.   

**Expected Result:**
Label should be defined for name edit field on project settings pane.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Suggested fix:** 
1- Use the title attribute. This technique is most often used when there is not space in the design for an explicit text label.  
e.g. 
<input type="text" title="Type search term here"/> <input type="submit" value="Search"/>  
 
2-Use the fieldset and legend elements to provide a description for groups of form elements. This is used for more complex forms and is often used in conjunction with other techniques. 
 
3-Use "aria-label". This technique is an alternative to the title attribute technique above in point 1.  
e.g. 
<button aria-label="Close" onclick="myDialog.close()">X</button> 
 
4-Use the "aria-describedby" property to provide a descriptive label for user interface controls 
e.g. 
<button aria-label="Close" aria-describedby="descriptionClose" 
    onclick="myDialog.close()">X</button> 
… 
<div id="descriptionClose">Closing this window will discard any information entered and 
return you back to the main page</div> 
5- Use 'aria-labelledby' to concatenate a label from several text nodes 
6-Providing text descriptions to identify required fields that were not completed 
 
**Key References for Fix:** 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html 

**Refer attachment file for more details:** 
![mas3 3 2_project settings_label is not defined for edit field](https://user-images.githubusercontent.com/30459830/28862103-6c04fef4-7781-11e7-924e-3653353f9525.png)

**Comment by v-mesala** [comment]
Bug is verified on URL :https://makecode.microbit.org/acc and bug is fixed hence closed the bug and added the closed tag.
